_data/jobs.yml

```diff
@@ -72,6 +76,12 @@
   jobs: []
 - company: nlc
   jobs: []
+- company: keen
+  jobs:
```

Comment:
looks like this has to be added in companies.yml, build is failing

Reply:
Thanks, seen. First time adding jobs.